### ISSUE TITLE
[Slider] Add support for secondary track

### ISSUE THREAD

**Issue by Tolriq** [material-components/material-components-android#1866]
**Is your feature request related to a problem? Please describe.**

I want to be able to show the cache buffer in a slider that allows to control a player. The value is informational and should not limit the ability to slide to values larger than the buffer.

**Describe the solution you'd like**
Behave like SeekBar and propose a secondary track to represent that value to users.

**Describe alternatives you've considered**
As all is private there's no way to implement that by extending the slider, alternative are seekbar with custom labels.

**Comment by wcshi** [comment]
Hi @Tolriq could you please share a mock of what you had in mind?

**Comment by Tolriq** [comment]
More or less like the normal android SeekBar/ProgressBar when using secondary Progress

https://developer.android.com/reference/android/widget/ProgressBar.html#attr_android:secondaryProgress

![image](https://user-images.githubusercontent.com/932609/98723495-c14a4700-2392-11eb-99f4-58d9a999e2d2.png)

But handling of the round corners and all the shiny material :)

**Comment by matpag** [comment]
This would be a nice feature to have. It would solve the Slider as "track player" use case.

**Comment by dovahkiin98** [comment]
It's a nice feature, but doesn't adding a Slider with a LinearProgressBar below it give the same behavior and look?

**Comment by Tolriq** [comment]
Not at all, proper positioning is complex due to rounding issues, it adds another view and most important it's drawn under so you always have the slider inactive track paint over when the wanted effect is the opposite.

**Comment by jimiduiveman-npo** [comment]
This feature would make sense indeed, what are the latest findings on this request?

**Comment by MusabAlothman** [comment]
Agree, feature is needed

**Comment by Ygor8686** [comment]
### 

> ### _****_

**Comment by Ygor8686** [comment]
I'm an .com.droit'
